Modules/Loadable/Markups/Widgets/Resources/UI/qSlicerMarkupsPlaceWidget.ui

```diff
@@ -196,10 +196,10 @@
      <normalon>:/Icons/Small/SlicerPointNumberLock.png</normalon>:/Icons/Small/SlicerPointNumberUnlock.png</iconset>
    </property>
    <property name="text">
-    <string>Point Number Locked</string>
+    <string>Control point number locked</string>
    </property>
    <property name="toolTip">
-    <string>Toggle markup point number lock</string>
+    <string>Toggle control point number lock</string>
```

Comment:
```suggestion
    <string>Toggle control point number lock. If locked then it is not possible to add or delete control points. Instead of deleting, position control points can be unset.</string>
```